index.html

```diff
@@ -623,6 +623,10 @@ <h2>
 
           <!-- Animation -->
           <div data-content="animation">
+            <div class="animation-preview-slider" id="animation-preview">
```

Comment:
https://github.com/Dun-sin/Code-Magic/blob/de52cb48bd6f37af313c1f1c87843ce2b517ef7f/src/style.css#L276

changing the classname to preview-slider saves you from having to style it from scratch.

since i already created the styling